### PR TITLE
cisco_aironet: add ECS mapping for event.severity

### PR DESCRIPTION
**Pull request by zmoog**
## Proposed commit message

Add ECS mapping for the `event.severity` field.

If a field has a string value, Elasticsearch maps it as a `keyword`. This happens even on stack versions 8.13+ because ecs@mappings does not perform type coercion.

According to the `sample_event.json` file, the `event.severity` field has a string value, so Elasticsearch maps it as a `keyword` instead of a `long`:

```json
    "event": {
        "action": "ENTRY_DELETED",
        "agent_id_status": "verified",
        "dataset": "cisco_aironet.log",
        "ingested": "2023-09-25T17:34:47Z",
        "original": "<134>WLC001: *SISF BT Process: Aug 20 11:25:50.157: %SISF-6-ENTRY_DELETED: sisf_shim_utils.c:482 Entry deleted A=fe80::aee2:d3ff:feba:56a4 V=0 I=wired:1 P=0000 M=",
        "provider": "SISF",
        "severity": "6",
        "timezone": "+00:00"
    }
```

By adding the `event.severity` field to the file `fields/ecs.yml` we ensure it has the expected ECS mapping. 

IMPORTANT: To fully resolve the issue, the input/integration owner should update it so that they can use the expected value to leverage ecs@mappings.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates https://github.com/elastic/integrations/issues/10848

### CODE REVIEW

File: packages/cisco_aironet/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.1"
+  changes:
+    - description: Fix the event.severity ECS field mapping.
+      type: enhancement
```
Comment (andrewkroh) [inline packages/cisco_aironet/changelog.yml:5]:
```suggestion
      type: bugfix
```

File: packages/cisco_aironet/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.1"
+  changes:
+    - description: Fix the event.severity ECS field mapping.
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/11105
```
Comment (taylor-swanson) [inline packages/cisco_aironet/changelog.yml:6]: With the merge of #11103, this will need to be bumped to `1.14.2`

**Review by taylor-swanson — COMMENTED** [review]
The pipeline test is failing because it is expecting `event.severity` to be a `long` but is currently a string. We could either add a type annotation to the grok or add a convert processor. I don't have strong opinions either way.

```
  - grok:
      field: _temp_.full_message
      ignore_failure: true
      patterns:
        - "%{DATA:event.provider}-%{INT:event.severity:long}-%{DATA:event.action}: %{DATA}:%{INT} %{GREEDYDATA:message}"
        - "%{DATA:event.provider}-%{INT:event.severity:long}-%{DATA:event.action}: %{GREEDYDATA:message}"
```

Once that change is made, sample_event and readme also have to be regenerated.

**Comment by andrewkroh** [comment]
@taylor-swanson Can you please take another look. I addressed your comment in b0f92da.

**Review by taylor-swanson — APPROVED** [review]

**Pull request merged**